openedx/features/caliper_tracking/transformers/enrollment_transformers.py

```diff
@@ -0,0 +1,21 @@
+"""
+Transformers for all the enrollment events
+"""
+
+
+def edx_course_enrollment_deactivated(current_event, caliper_event):
+    """
+    Caliper specific log for edX course unenrollment
+    """
+    caliper_object = {
+        'course_id': caliper_event['course_id'],
```

Comment:
since `course_id` is removed from base transformer, this should be updated link `'course_id': current_event['context']['course_id'],`

Reply:
done